src/components/Common/CaseStudyCards/CaseStudyBlocks.js

```diff
@@ -0,0 +1,172 @@
+      bpColorReverse ? theme.colors.white : null};
+    font-family: 'Roboto', sans-serif;
+  }
+`
+
+const renderText = (
+  text,
+  colorReverse,
+  bpColorReverse = false,
+  disallowed = [],
+  bpFont = null
+) =>
+  text && (
+    <ReactMarkdown
+      disallowedTypes={disallowed}
+      renderers={{
+        // eslint-disable-next-line
+        heading: props => <SectionTitle reverse={colorReverse} {...props} />,
+        // eslint-disable-next-line
+        paragraph: props => (
+          <StyledBodyPrimary
+            reverse={colorReverse}
+            bpColorReverse={bpColorReverse}
+            bpFont={bpFont}
+            {...props}
+          />
+        )
+      }}
+      source={text}
+    />
+  )
+
+const MobileReverseOrderWrapper = styled.div`
+  display: flex;
+  flex-direction: column-reverse;
+
+  ${breakpoint('smallTablet')`
+    flex-direction: column;
+  `}
+`
+
+const TextAndImagePaddingSeparator = styled.div`
+  padding-bottom: ${({ theme }) => theme.space[2]};
+`
+
+const TextColumnsBlock = ({
+  data: { text },
+  colWidthOne = [1, 1, 1, 1, 1 / 2],
+  colWidthTwo = [1, 1, 1, 1, 1 / 2],
+  middleColWidth = null,
+  colorReverse = false,
+  bpColorReverse = false
+}) => (
+  <Fragment>
+    <Col width={colWidthOne}>
+      {renderText(text, colorReverse, bpColorReverse, ['paragraph'])}
+    </Col>
+    {middleColWidth && <Col width={middleColWidth} />}
+    <Col width={colWidthTwo}>
+      {renderText(text, colorReverse, bpColorReverse, ['heading'])}
+    </Col>
+  </Fragment>
+)
+
+const FullWidthBlock = ({ data: { text, image } }) => (
+  <Col width={[1]}>
+    {renderText(text)}
+    {text && image && <TextAndImagePaddingSeparator />}
+    {renderImage(image)}
+  </Col>
+)
+
+const HalfGreyBackground = styled(GreyBackground)`
+  display: ${({ mobile }) => (mobile ? 'inherit' : 'none')};
+  background-image: linear-gradient(white 50%, transparent 50%);
+
+  ${breakpoint('smallTablet')`
+      display: ${({ mobile }) => (mobile ? 'none' : 'inherit')};
+  `}
+`
+
+const VideoBlock = ({ data: { text } }) => <VideoSection src={text} />
+
+const ImagesBlock = ({ data }) => (
+  <Fragment>
+    {data.map((image, index) => (
+      <StyledColumnImage
+        width={[2 / 5, 2 / 5, 1 / 5, 1 / 5, 1 / 4, 1 / 4]}
+        key={index}
+      >
+        {renderImage(image)}
+      </StyledColumnImage>
+    ))}
+  </Fragment>
+)
+
+const StyledMobilePadding = styled(Col)`
+  padding-bottom: ${({ theme }) => theme.space[4]};
+`
+
+const StyledBreakpointMobilePadding = styled(StyledMobilePadding)`
+  ${breakpoint('smallTablet')`
+    display: none;
+  `}
+`
+
+const TextAndImageBlock = ({
+  data: { text, image },
+  colWidthOne = [1, 1, 1, 1, 1 / 2],
+  colWidthTwo = [1, 1, 1, 1, 1 / 2],
+  middleColWidth = null,
+  colorReverse = false
+}) => (
+  <Fragment>
+    <Col width={colWidthOne}>{renderText(text, colorReverse)}</Col>
+    {text && image && <StyledBreakpointMobilePadding width={[1]} />}
```

Comment:
Can you not just  pass `width={[0, 0, 0, 1]}` to here? It'll go to the the `Col` as props

Reply:
Oh yeah! That works too! didn't think of that!